core-data/src/main/kotlin/io/getstream/avengerschat/core/data/repository/dm/DirectMessageRepositoryImpl.kt

```diff
@@ -61,7 +61,7 @@ internal class DirectMessageRepositoryImpl @Inject constructor(
     )
     val result = chatClient.queryUsers(usersRequest).await()
     result.onSuccessSuspend {
-      emit(result.data())
+      emit(it)
```

Comment:
`Result` class has been improved and extracted to a different artifact.
It improves how it is used taking advantage of Kotlin language.